cmd/node/main.go

```diff
@@ -590,8 +590,15 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 
 	log.Debug("config", "file", ctx.GlobalString(genesisFile.Name))
 
+	exportFolder := filepath.Join(workingDir, generalConfig.Hardfork.ImportFolder)
+	nodesSetupPath := ctx.GlobalString(nodesFile.Name)
+	if generalConfig.Hardfork.AfterHardFork {
+		if core.DoesFileExist(filepath.Join(exportFolder, "nodesSetup.json")) {
```

Comment:
No. It is not possible to have it. If we do a hardfork afterHardfork must be set to true.

Reply:
I will use a variable for the file's name. 
It is not ok if the file does not exist after hard fork, but instead of an error because the file was not found, I preferred to use the genesis config one. I can make the node not to start if the `nodesSetup.json` file does not exist. As I know, we can't do a "partial" hardfork